lxd/networks.go

```diff
@@ -1786,6 +1808,12 @@ func networkStartup(stateFunc func() *state.State, restoreOnly bool) error {
 			for pn := range initNetworks[priority] {
 				err := loadAndInitNetwork(s, pn, priority, true, restoreOnly)
 				if err != nil {
+					// When restoring a network the operation is not allowed to fail.
+					// The network is already started at this stage which might have taken multiple retries.
```

Comment:
@roosterfish can it happen that LXD is still trying to start up a network in a loop in a different thread (started during LXD start up), and then `lxc cluster restore` is run before the local network is started, and then this will call `n.Restore()` before the network itself is started?

Reply:
or if thats going to be too invasive to reason about at this time, then we can at least run `d.db.Cluster.LocalNodeIsEvacuated()` internally for each iteration here?

Reply:
Sounds good thanks!